src/main/io/displayport_msp.c

```diff
@@ -229,4 +223,8 @@ displayPort_t *displayPortMspInit(void)
 void displayPortMspSetSerial(serialPortIdentifier_e serialPort) {
     displayPortSerial = serialPort;
 }
+
+serialPortIdentifier_e displayPortMspGetSerial(void) {
```

Comment:
(maybe `displayportIsUsingPort(mspPort_t * port) / displayportIsEnabledOnPort(mspPort_t * port))` or similar name - moving the internals to displayport_msp.c))

Reply:
Many ways to skin a cat. Both `msp_serial.c` and `displayport_msp.c` are a legitimate place to have MSP coding knowledge, one might think given the filenames, but I'd argue that `msp_serial.c` has understanding of MSP, whereas `displayport_msp.c` only understands display port encodings so having the latter understand `mspPort_t` is tipping the balance the wrong way. Both use `serialPortIdentifier_e` however, so I'll leave this as is.

Reply:
(the idea was to hide the ugly part closer to source. Otherwise your argument is valid and preferred)